kernel-api/src/main/scala/org/apache/toree/dependencies/IvyDependencyDownloader.scala

```diff
@@ -87,7 +88,7 @@ class IvyDependencyDownloader(repositoryUrl: String, baseDirectory: String)
     ivyFile.deleteOnExit()
 
     val md = DefaultModuleDescriptor.newDefaultInstance(
-      ModuleRevisionId.newInstance("com.ibm.spark", "spark-kernel", "working")
+      ModuleRevisionId.newInstance("org.apache.toree", "spark-kernel", "working")
```

Comment:
This shouldn't be a problem. We are generating a new XML file that Ivy will use and had to create an overall name for it (org, artifact name, version, etc). Hence the `working` bit of it. You can change it to `toree-kernel` to be consistent if you want.